src/main/java/org/apache/sling/feature/maven/mojos/AnalyseFeaturesMojo.java

```diff
@@ -98,6 +99,20 @@ public URL provide(final ArtifactId id) {
         }
         getLog().debug(MessageUtils.buffer().strong("Scanner").a(" successfully set up").toString());
 
+        FeatureProvider featureProvider = new FeatureProvider() {
```

Comment:
It would be good to use new BaseFeatureProvider() from the AbstractFeatureMojo class

Reply:
Thanks for the suggestion, I've made this change now.